client/src/sass/CheckIn.scss

```diff
@@ -147,6 +146,12 @@ input[type=radio] {
             background-color: black;
             color: white;
         }
+        &:disabled {
```

Comment:
Good to know. Cool trick and way to use css. https://css-tricks.com/the-sass-ampersand/. 🙇